include/color_spinor.h

```diff
@@ -448,9 +448,9 @@ namespace quda {
 		    -i  0  0  0
 		     0  i  0  0
     */
-    __device__ __host__ inline ColorSpinor<Float,Nc,4> sigma(int mu, int nu) {
+    __device__ __host__ inline ColorSpinor<Float,Nc,4> sigma(int mu, int nu) const {
       ColorSpinor<Float,Nc,4> a;
```

Comment:
Any particular reason these are (broadly) `4` as opposed to defining an `Ns` somewhere? I can see a case for keeping the `4` explicit.

Reply:
(this is not new code, and has been around for years) 

The intention here was to clearly show that these members are in the `Ns=4` specialized class.